### DISCUSSION TITLE
How to configure file associations for Windows?

### DISCUSSION THREAD

**Discussion by eggbean** [gokcehan/lf#1578] [Q&A] [answered]
On Linux everything seems to work with no configuration, because of `xdg-open` I assume, but how do I configure this on Windows?

I have tried looking for information, but there doesn't seem to be much for Windows and what I have involved installing a lot of python stuff which I don't want to deal with.

**Comment by joelim-work** [comment, marked as answer]
For Windows `start` is used instead of `xdg-open`: https://github.com/gokcehan/lf/blob/master/doc.md#opener

I think you just have to change the default application for opening the file with, and `lf` should use it too.

**Comment by eggbean** [comment]
Oh, it's working without me having to have done anything now. It wasn't before. Thanks.
